src/main/java/mods/railcraft/common/blocks/machine/manipulator/TileFluidLoader.java

```diff
@@ -54,7 +52,7 @@ public ManipulatorVariant getMachineType() {
     }
 
     @Override
-    public EnumFacing getFacing() {
+    public EnumFacing getDefaultFacing() {
```

Comment:
Probably shouldn't change this, fluid loaders can't rotate.

At least point getFacing() at getDefaultFacing().

Reply:
many places access the facing field instead of calling this method. as a result, it's safer to initialize the facing field to default facing and limit rotations instead.

Reply:
Had to fix this to make it compile against my changes to ITileRotate.